docs/docs/en/guide/installation/kubernetes.md

```diff
@@ -258,24 +269,29 @@ docker build -t apache/dolphinscheduler:mysql-driver .
 
 > Because of the commercial license, we cannot directly use the driver of Oracle.
 >
-> If you want to add Oracle datasource, you can build a new image based on the `apache/dolphinscheduler` image follow the following instructions:
+> If you want to add Oracle datasource, you can build a new image based on the `apache/dolphinscheduler-<service>` image follow the following instructions:
+>
+> You needs to change the two images of dolphinscheduler-worker dolphinscheduler-api service.
```

Comment:
could we combine MySQL and Oracle into the same section? I find out the only different is jar name, WDYT?

Reply:
Sure